pysb/importers/sbml.py

```diff
@@ -0,0 +1,129 @@
+from __future__ import print_function as _
+from pysb.importers.bngl import model_from_bngl
+from pysb.bng import _get_bng_path
+import subprocess
+import os
+import tempfile
+import shutil
+
+
+class SbmlTranslationError(Exception):
+    pass
+
+
+def sbml_translator(input_file,
+                    output_file=None,
+                    convention_file=None,
+                    naming_conventions=None,
+                    user_structures=None,
+                    molecule_id=False,
+                    atomize=False,
+                    pathway_commons=False,
+                    verbose=False):
+    """
+    Runs the BioNetGen sbmlTranslator binary. For more descriptions of
```

Comment:
Keep summary to one line, and reformat params section using numpydoc/napoleon conventions.